broker/broker-undelivered-messages-ui/src/main/webapp/index.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import React from "react"
+import {render} from "react-dom"
+import store from './store'
+import {
+    isAllSelected,
+    displayNumSelected,
+    getSelectedIds,
+    getSelectedMessages,
+    getIds
+} from './reducer'
+import {
+    addMessage,
+    checkAllMessages,
+    togglePolling,
+    expandAllMessages,
+    updateDeleted,
+    removeMessages
+} from './actions'
+import TableView from './table.view'
+
+require('es6-promise').polyfill()
+require('isomorphic-fetch')
+
+const getMessagesURI = '/admin/jolokia/exec/mil.af.gdes.message.broker.UndeliveredMessages:service=UndeliveredMessages/getMessages/jms.queue.DLQ/Core/'
+
+const Json = ({value}) => (
+    <pre>{JSON.stringify(value, null, 2)}</pre>
+)
+
+const retrieveData = (data, url) => {
+    return window.fetch(url, {credentials: 'same-origin'})
+        .then((res) => res.json())
+        .then((json) => {
+                json.value.filter((message) => !(getIds(data).includes(message.messageID))).map((message) => {
+                    store.dispatch(addMessage({
+                        timestamp: message.timestamp,
+                        id: message.messageID,
+                        expanded: false,
+                        message: {
+                            messageBody: message.body,
+                            origin: message.StringProperties._AMQ_ORIG_QUEUE,
+                            address: message.address
+                        }
+                    }))
+                })
+            }
+        )
+}
+
+const operateOnData = (data, method) => {
+    var ids = (method === 'resendMessages') ? getSelectedIds(getSelectedMessages(data).filter((message) => message.message.origin.length !== 0)) : getSelectedIds(data)
+    window.fetch("/admin/jolokia/", {
+        method: "POST",
+        credentials: "same-origin",
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+            type: "EXEC",
+            mbean: "mil.af.gdes.message.broker.UndeliveredMessages:service=UndeliveredMessages",
+            operation: method,
+            arguments: ["jms.queue.DLQ", "Core", ids]
+        })
+    })
+        .then((res) => {
+            var x = res.json()
+            console.log(x)
```

Comment:
I think we were using this for debugging. I think we can collapse it to `(res) => res.json()` now.